SeisSeeker/processing/detection.py

```diff
@@ -1202,7 +1214,9 @@ def detect_events(self, verbosity=0):
                 # plt.gca().yaxis.set_major_locator(MaxNLocator(5)) 
                 for i in range(3):
                     ax[i].xaxis.set_major_locator(plt.MaxNLocator(3))
+                fig.savefig(f'{self.outdir}/plots/detection_t_series/Phase_assocaition_{f_uid}.png', dpi=600)
```

Comment:
Sorry to be pedantic. Would it be ok to use os.path.join() for paths rather than "/". As then the code is OS independent.